deployment/kube/prod/hpa.yaml

```diff
@@ -12,7 +12,7 @@ spec:
     kind: Deployment
     name: imageserver
   # Set this to 3x "min-nodes":
-  minReplicas: 12
-  # Set this to 3x "max-nodes":
+  minReplicas: 3
+  # Set this to 12x "max-nodes":
```

Comment:
Can you please explain the reason for this change?

Reply:
this is to scale down we want to have a minimum a 3 instances when the load is low.

Reply:
I mean changing `maxReplicas` to be 12x "max-nodes".

Reply:
i might have to increase it, we peaked at 16 last night

Reply:
Why are we changing this from 3x "max-nodes" to 12x or 16x "max-nodes"? Unless something else changed, the "max-nodes" variable set in GKE sets the maximum number of nodes per zone, and so with 3 zones we want to multiply this number by 3 to get "maxReplicas".

"minReplicas" works the same way except for "min-nodes". 

If things have changed (as a result of some change elsewhere in GKE, or as a result of your work), please explain what's changed.